std/typecons.d

```diff
@@ -282,6 +283,51 @@ private:
 
 @system unittest
 {
+    debug(Unique) writeln("Unique interface");
```

Comment:
Good. I was going to request that they be removed but it's easy to submit a PR removing them all at once so I didn't bother.